packages/11ty/content/_assets/javascript/application/canvas-panel.js

```diff
@@ -1,51 +1,107 @@
-window.onload = function () {
-  const canvasPanels = document.getElementsByTagName('canvas-panel')
+import scrollToHash from "./scroll-to-hash"
+
+/**
+ * Handle annotation/choice selection with canvasPanel API
+ * {@link https://iiif-canvas-panel.netlify.app/docs/examples/handling-choice}
+ */
+const handleSelect = (element) => {
+  const figure = element.closest('.q-figure')
+  if (!figure) return
+  const canvasPanel = figure.querySelector('canvas-panel')
+  const annotationType = element.getAttribute('data-annotation-type')
+  const id = element.getAttribute('id')
+  const inputType = element.getAttribute('type')
```

Comment:
Does it makes sense to destructure `element.attributes` for these values?

Reply:
not sure how to do this in a way that is not very verbose, unless you have a suggestion?